src/widget/text.rs

```diff
@@ -0,0 +1,40 @@
+use super::Widget;
+use embedded_graphics::{mono_font::MonoTextStyle, prelude::*, text};
+
+#[derive(Clone, Copy)]
+pub struct Options<Color> {
+    pub character_style: MonoTextStyle<'static, Color>,
```

Comment:
In this context is `MonoTextStyle` really ok with `Color` being _anything_ (since there aren't any constraints on the type of `Color` here)?

Reply:
Right, I could add a constraint on it to the struct definition to enforce that, but in practice I think that's kind of boiler-platey, since you still need to repeat it on the `impl`, and doesn't really get you much benefit.